financialaid/api.py

```diff
@@ -0,0 +1,33 @@
+"""
+API helper functions for financialaid
+"""
+from financialaid.constants import COUNTRY_INCOME_THRESHOLDS, DEFAULT_INCOME_THRESHOLD
+
+
+def determine_tier_program(program, income):
+    """
+    Determines and returns the TierProgram for a given income.
+    Args:
+        program: Program
+        income: numeric
+    Returns:
+        TierProgram
```

Comment:
Args and Returns should include the type and a brief description.
See [this](https://github.com/mitodl/micromasters/blob/master/mail/api.py#L47-L52) as an example